Sources/Starknet/Accounts/StarknetAccount.swift

```diff
@@ -30,44 +24,67 @@ public class StarknetAccount: StarknetAccountProtocol {
         self.cairoVersion = cairoVersion
     }
 
-    private func makeInvokeTransaction(calldata: StarknetCalldata, signature: StarknetSignature, params: StarknetExecutionParams, forFeeEstimation: Bool = false) -> StarknetInvokeTransactionV1 {
+    private func makeInvokeTransactionV1(calldata: StarknetCalldata, signature: StarknetSignature, params: StarknetDeprecatedExecutionParams, forFeeEstimation: Bool = false) -> StarknetInvokeTransactionV1 {
         StarknetInvokeTransactionV1(senderAddress: address, calldata: calldata, signature: signature, maxFee: params.maxFee, nonce: params.nonce, forFeeEstimation: forFeeEstimation)
     }
 
-    private func makeDeployAccountTransaction(classHash: Felt, salt: Felt, calldata: StarknetCalldata, signature: StarknetSignature, params: StarknetExecutionParams) -> StarknetDeployAccountTransactionV1 {
-        StarknetDeployAccountTransactionV1(
-            signature: signature,
-            maxFee: params.maxFee,
-            nonce: params.nonce,
-            contractAddressSalt: salt,
-            constructorCalldata: calldata,
-            classHash: classHash
-        )
+    private func makeInvokeTransactionV3(calldata: StarknetCalldata, signature: StarknetSignature, params: StarknetExecutionParamsV3, forFeeEstimation: Bool = false) -> StarknetInvokeTransactionV3 {
+        StarknetInvokeTransactionV3(senderAddress: address, calldata: calldata, signature: signature, l1ResourceBounds: params.resourceBounds.l1Gas, nonce: params.nonce, forFeeEstimation: forFeeEstimation)
```

Comment:
nit: What's the rationale behind prefixing all these names - `StarknetInvokeTransactionV3`, `StarknetExecutionParamsV3`, etc. with `Starknet`?

Reply:
It was a [convention](https://github.com/software-mansion/starknet.swift/pull/63#discussion_r1124471672) before I join, so I cannot fully vouch, but as I understood, it was done to prevent namespace pollution. Unlike in other languages, it's typical to import entire lib.